flink-test-utils-parent/flink-test-utils-junit/src/main/java/org/apache/flink/util/DockerImageVersions.java

```diff
@@ -33,7 +33,7 @@
     public static final String ELASTICSEARCH_6 =
             "docker.elastic.co/elasticsearch/elasticsearch-oss:6.3.1";
 
-    public static final String KAFKA = "confluentinc/cp-kafka:5.5.2";
+    public static final String KAFKA = "confluentinc/cp-kafka:6.2.1";
```

Comment:
Shouldn't we also update https://github.com/fapaul/flink/blob/FLINK-24765/flink-end-to-end-tests/flink-end-to-end-tests-common-kafka/src/test/java/org/apache/flink/tests/util/kafka/KafkaSourceE2ECase.java#L37 and https://github.com/fapaul/flink/blob/FLINK-24765/flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/connector/kafka/source/KafkaSourceITCase.java#L248 ?

Reply:
I've already run into this in the past, would it be possible to unify these (a single constant shared between these tests)?